src/Html/HtmlAttributesTrait.php

```diff
@@ -242,10 +241,14 @@ private function _removeClass(string $class)
   /**
    * Retrieve all classes set on the element
    *
-   * @return string[]
+   * @return array<string>
```

Comment:
```suggestion
   * @return string[]
```
this was correct before. typed array notation is only really useful if you need to specify key type or multi-dimensional arrays